src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/StatusIcon.tsx

```diff
@@ -1,93 +1,171 @@
 import React from 'react'
-import { Icon } from 'react-feather'
-import styled, { css } from 'styled-components/macro'
+import styled from 'styled-components/macro'
+import { transparentize } from 'polished'
+import SVG from 'react-inlinesvg'
 
 export type StatusIconState = 'success' | 'pending' | 'not-started' | 'error'
 
-interface StepIconStyle {
-  bgColor: string
-  color: string
-  borderColor: string
-}
-
-const STATUS_ICON_STYLES: Record<StatusIconState, StepIconStyle> = {
-  success: {
-    bgColor: '#D5E5E3',
-    borderColor: '#D5E5E3',
-    color: '#017B28',
-  },
-  pending: {
-    bgColor: '#F8F9FD',
-    borderColor: '#F8F9FD',
-    color: '#0D5ED9',
-  },
-  'not-started': {
-    bgColor: 'none',
-    borderColor: '#D6DDE9',
-    color: '#D6DDE9',
-  },
-  error: {
-    bgColor: '#f25757',
-    borderColor: '#f25757',
-    color: 'white',
-  },
-}
-
-const LABEL_COLOR: Record<StatusIconState, string> = {
-  success: STATUS_ICON_STYLES['success'].color,
-  pending: STATUS_ICON_STYLES['pending'].color,
-  'not-started': 'gray',
-  error: STATUS_ICON_STYLES['error'].bgColor,
-}
+const Content = styled.span`
+  display: flex;
+  flex-flow: column wrap;
+  gap: 6px;
+`
 
-const StepIcon = styled.div<StepIconStyle>`
-  border-radius: 50%;
-  width: 40px;
-  height: 40px;
+const StepIcon = styled.div<{ status: StatusIconState }>`
+  --size: 52px;
+  border-radius: var(--size);
+  width: var(--size);
+  height: var(--size);
   display: flex;
   justify-content: center;
   align-items: center;
-  margin-bottom: 5px;
-  border: 2px solid;
+  margin: 0 0 6px;
+  border: ${({ status, theme }) =>
+    status === 'pending'
+      ? 'none'
```

Comment:
combining the ternary condition like `status === ('pending' || 'error')` did not seem to work for some reason.

Reply:
That doesn't work, you have to do 2 different comparisons like: `status === 'pending || status === 'error'`